ui/src/components/admin/forms/AdminFormDataViewer.js

```diff
@@ -401,7 +496,11 @@ export default function AdminFormDataViewer(props) {
         columnDefs.push(createNumberColumn(key));
       } else if (typeof value === 'boolean') {
         columnDefs.push(createBooleanColumn(key));
-      } else {
+      } else if (key === 'baptismInfo'){
```

Comment:
this wouldn't work. have you tried loading a form where the first entry/user does not have baptism/membership info? (e.g. Life Defined form in UAT). i don't think baptism/membership columns' visibility should depend on whether the first user in the data has baptism/membership info. probably should just hard code these 2 columns.